package.json

```diff
@@ -4,12 +4,22 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "build-bundle": "vue-cli-service build --target lib --name modeler ./src/components/nodes/index.js",
+    "open-cypress": "TZ=UTC cypress open",
     "lint": "vue-cli-service lint --no-fix",
+    "build-bundle": "vue-cli-service build --target lib --name modeler ./src/components/nodes/index.js",
     "lint-fix": "vue-cli-service lint",
-    "test": "TZ=UTC cypress open",
-    "test-processmaker": "cypress open --env inProcessmaker=true",
-    "test-ci": "TZ=UTC vue-cli-service test:e2e --mode test --headless --browser chrome"
+    "pretest": "rm -rf .nyc_output || true",
+    "test": "npm run test-unit && npm run test-ci",
```

Comment:
Oh, the readme does say this. It may need to be updated as well.

```
# Run the Cypress end-to-end (e2e) test suite
npm test

# Run the Cypress end-to-end (e2e) test suite in headless mode
npm run test-ci
```

Reply:
Good catch! I'll fix it right now.